tests/common/mod.rs

```diff
@@ -8,166 +9,229 @@ pub fn enable_mockito_logging() {
     env_logger::init();
 }
 
-pub fn create_bridge(status_code: usize, body: &str) -> (Mock, Bridge) {
-    create_bridge_with_path(status_code, body, "/")
+#[async_trait]
+pub trait MockitoServerCreateBridgeExt {
```

Comment:
I was looking at the tests but I don't see where this is used :thinking:

Reply:
It's typically imported and used by the async_bridge rest tests:
https://github.com/primait/bridge.rs/blob/c99d6dd5cd369814b772e7b2add87ce36bd2a01b/tests/async_bridge/rest.rs#L9

Reply:
Completely missed that, thanks